cmd/networkPolicies.go

```diff
@@ -3,35 +3,82 @@ package cmd
 import (
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	networking "k8s.io/api/networking/v1"
 )
 
-func checkNamespaceNetworkPolicies(netPols *NetworkPolicyList) {
-	badNetPols := []NetworkPolicy{}
+func getNamespacesMap(namespaceList *NamespaceList) map[string]bool {
+	// TODO filter namespaces based on annotations
+	nsMap := map[string]bool{}
+
+	for _, ns := range namespaceList.Items {
+		nsMap[ns.Name] = false
+	}
+
+	return nsMap
+}
+
+func printResult(auditedNamspaces map[string]bool) {
```

Comment:
I agree with you. I will take a look at how I can use the `runAudit` function and `Results` (in the long run this will also make the test cases simpler). I you like one "big" PR I can integrate the changes into this PR otherwise I would create a second one, depends on your choice :)